src/main/java/org/terasology/commonworld/heightmap/CachingHeightMap.java

```diff
@@ -28,35 +15,35 @@
     private static final Logger logger = LoggerFactory.getLogger(CachingHeightMap.class);
 
     private final short[] height;
-    private final Rect2i area;
+    private final Rectanglei area;
```

Comment:
We could potentially use `BlockArea` here. But let's address this when doing an overhaul of the whole cities code.